DirectProgramming/DPC++/StructuredGrids/iso2dfd_dpcpp/src/iso2dfd.cpp

```diff
@@ -304,14 +304,14 @@ int main(int argc, char* argv[]) {
         //    swaps their content at every iteration.
         if (k % 2 == 0)
           h.parallel_for(global_range, [=](id<2> it) {
```

Comment:
in parallel_for function use "auto" for id as per the guidelines

Reply:
Code updated. Thanks for the suggestion